internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -45,6 +51,7 @@ var (
 	errUIDWithNonManagedFS = errors.New("UID and GID cannot be specified with non-managed EFS")
 	errInvalidUIDGIDConfig = errors.New("set managed filesystem access point creation info: must specify both UID and GID, or neither")
 	errReservedUID         = errors.New("set managed filesystem access point creation info: UID must not be 0")
+	errInvalidSpotConfig   = errors.New(`invalid spot configuration`)
```

Comment:
Can we be more specific about the reason why the spot configuration is invalid? So it could be fixed more easily.